packages/frontend/web-ui/src/app/store/thunk/createAuthByRefreshToken.ts

```diff
@@ -0,0 +1,31 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import { buildSerializableResponse } from '../../../common/http/helpers/buildSerializableResponse';
+import { AuthResponse } from '../../../common/http/models/AuthResponse';
+import { AuthSerializedResponse } from '../../../common/http/models/AuthSerializedResponse';
+import { httpClient } from '../../../common/http/services/HttpService';
+import { useAppSelector } from '../hooks';
+import { selectAuthenticatedAuth } from './../features/authSlice';
+
+export const createAuthByRefreshToken = createAsyncThunk(
+  'auth/createAuthByRefreshToken',
+  async (): Promise<AuthSerializedResponse> => {
```

Comment:
I would add a `refreshToken` parameter here, I don't know what should we do if there's no token.